x-pack/plugins/ml/public/application/app.tsx

```diff
@@ -45,6 +45,12 @@ interface AppProps {
 
 const localStorage = new Storage(window.localStorage);
 
+// temporary function to hardcode the serverless state
+// this will be replaced by the true serverless information from kibana
+export function isServerless() {
+  return true;
```

Comment:
This should be set to `false` before merging shouldn't it?

Reply:
yes!

Reply:
https://github.com/elastic/kibana/pull/149419/commits/35e95a5e382fc5571aac877f2390a65a2c680000